src/config/unit_test.rs

```diff
@@ -1447,4 +1447,51 @@ mod tests {
                     ]);
                 */
     }
+
+    #[tokio::test]
+    async fn test_expand() {
```

Comment:
I think this test would better fit into `tests/behavior/transform/`. I think the tests in this file are more about testing unit test configuration than validating transforms.

Reply:
I never looked at those tests. This make much more sense to put it there. Will relocate.